ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/CompactorTest.java

```diff
@@ -401,7 +401,7 @@ private void addFile(Table t, Partition p, long minTxn, long maxTxn, int numReco
     switch (type) {
       case BASE: filename = AcidUtils.BASE_PREFIX + maxTxn + (visibilityId > 0 ? AcidUtils.VISIBILITY_PREFIX + visibilityId : ""); break;
       case LENGTH_FILE: // Fall through to delta
-      case DELTA: filename = makeDeltaDirName(minTxn, maxTxn); break;
+      case DELTA: filename = AcidUtils.addVisibilitySuffix(makeDeltaDirName(minTxn, maxTxn),visibilityId); break;
```

Comment:
This still isn't 100% true-to-life, since only compacted deltas have a visibility suffix, FWIW. Maybe could add a COMPACTED_DELTA type?

Reply:
I followed the same approach as for the BASE folder: if we won't provide `visibilityId` - 0 value would be passed instead and the final path would be for uncompacted data.
Renaming of base methods (addBase/DeltaFile => addCompactedStuff) leads to multiple file changes. I think it's better to address this in a separate `refactoring `JIRA.